src/pages/auth/register.tsx

```diff
@@ -38,9 +38,13 @@ const Register = () => {
   const handleSubmit = (e: React.ChangeEvent<HTMLInputElement>) => {
     e.preventDefault();
 
-    emailValidation(email, setEmailInvalid);
-    passwordValidation(password, setPasswordInvalid);
-    userNameValidation(userName, setuserNameInvalid);
+    if (emailValidation(email) === false) {
+      setEmailInvalid(true);
+    }
+    if (passwordValidation(password) === false) {
+      setPasswordInvalid(true);
+    }
+    if (userNameValidation(userName) === false) setuserNameInvalid(true);
```

Comment:
```suggestion
    if (!userNameValidation(userName)){
       setuserNameInvalid(true);
    } 
```

What is it ?

We need to follow the default code style. Fix this for better code readability and easier maintenance